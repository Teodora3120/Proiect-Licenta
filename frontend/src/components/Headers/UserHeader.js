/*!

=========================================================
* Argon Dashboard React - v1.2.2
=========================================================

* Product Page: https://www.creative-tim.com/product/argon-dashboard-react
* Copyright 2022 Creative Tim (https://www.creative-tim.com)
* Licensed under MIT (https://github.com/creativetimofficial/argon-dashboard-react/blob/master/LICENSE.md)

* Coded by Creative Tim

=========================================================

* The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.

*/

// reactstrap components
import { Container, Row, Col } from 'reactstrap'

const UserHeader = (props) => {
  return (
    <>
      <div
        className="header pb-8 pt-5 pt-lg-8 d-flex align-items-center"
        style={{
          minHeight: '300px',
          backgroundImage:
            'url(' + require('../../assets/img/theme/profile-cover.jpg') + ')',
          backgroundSize: 'cover',
          backgroundPosition: 'center top',
        }}
      >
        {/* Mask */}
        <span className="mask bg-gradient-default opacity-8" />
        {/* Header container */}
        <Container className="d-flex align-items-center" fluid>
          <Row>
            <Col lg="7" md="10">
              <h1 className="display-2 text-white ws-0">Hello {props.firstName + " " + props.lastName}</h1>
              <p className="text-white mt-0 mb-5">
                This is your profile page. You can edit your information here.
              </p>
            </Col>
          </Row>
        </Container>
      </div>
    </>
  )
}

export default UserHeader
