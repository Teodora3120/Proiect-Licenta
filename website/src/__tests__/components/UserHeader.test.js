import React from 'react'
import { shallow } from 'enzyme'
import UserHeader from '../../components/Headers/UserHeader'
test('should test User Header component', () => {
  const wrapper = shallow(<UserHeader />)
  expect(wrapper).toMatchSnapshot()
})
