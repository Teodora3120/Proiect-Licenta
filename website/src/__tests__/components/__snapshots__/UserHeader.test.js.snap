// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test User Header component 1`] = `
<Fragment>
  <div
    className="header pb-8 pt-5 pt-lg-8 d-flex align-items-center"
    style={
      Object {
        "backgroundImage": "url(profile-cover.jpg)",
        "backgroundPosition": "center top",
        "backgroundSize": "cover",
        "minHeight": "300px",
      }
    }
  >
    <span
      className="mask bg-gradient-default opacity-8"
    />
    <Container
      className="d-flex align-items-center"
      fluid={true}
      tag="div"
    >
      <Row
        tag="div"
        widths={
          Array [
            "xs",
            "sm",
            "md",
            "lg",
            "xl",
          ]
        }
      >
        <Col
          lg="7"
          md="10"
          tag="div"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          <h1
            className="display-2 text-white ws-0"
          >
            Hello 
            undefined undefined
          </h1>
          <p
            className="text-white mt-0 mb-5"
          >
            This is your profile page. You can edit your information here.
          </p>
        </Col>
      </Row>
    </Container>
  </div>
</Fragment>
`;
